lib/perl/Genome/Interfaces/Comparable/Command/Diff.pm

```diff
@@ -58,6 +58,7 @@ sub diffs_message {
     my $self = shift;
     my $diffs = $self->_diffs;
     my $diff_string = "DIFFERENCES FOUND:\n";
+    $diff_string .= sprintf(" Comparing new object %s to blessed object %s\n", $self->new_object->id, $self->blessed_object->id);
```

Comment:
Yeah, the main thing is that this one will make it into the e-mail, whereas the other one is just a status message and will only be in the jenkins console log